packages/react/src/ProgressBar/ProgressBar.tsx

```diff
@@ -60,37 +57,76 @@ const ProgressContainer = styled.span<StyledProgressContainerProps>`
   ${sx};
 `
 
+export type ProgressBarItems = React.HTMLAttributes<HTMLSpanElement> & {'aria-label'?: string} & ProgressProp & SxProp
+
+export const Item = forwardRef<HTMLSpanElement, ProgressBarItems>(
+  (
+    {progress, 'aria-label': ariaLabel, 'aria-valuenow': ariaValueNow, 'aria-valuetext': ariaValueText, ...rest},
+    forwardRef,
+  ) => {
+    const progressAsNumber = typeof progress === 'string' ? parseInt(progress, 10) : progress
+
+    const ariaAttributes = {
+      'aria-valuenow':
+        ariaValueNow || (progressAsNumber !== undefined && progressAsNumber >= 0 ? Math.round(progressAsNumber) : 0),
+      'aria-valuemin': 0,
+      'aria-valuemax': 100,
+      'aria-valuetext': ariaValueText,
+    }
+
+    return (
+      <ProgressItem
+        role="progressbar"
+        aria-label={ariaLabel}
+        ref={forwardRef}
+        progress={progress}
+        {...ariaAttributes}
+        {...rest}
```

Comment:
One suggestion to prevent accidental overrides of earlier props if that'd be helpful. This showed up recently for `className` specifically so just wanted to share 👍 

```suggestion
        {...rest}
        role="progressbar"
        aria-label={ariaLabel}
        ref={forwardRef}
        progress={progress}
        {...ariaAttributes}
```

Reply:
Interesting! Thank you for the suggestion!